common/common.go

```diff
@@ -0,0 +1,14 @@
+// Package common holds the common types used across the node
+
+package common
+
+import "github.com/ethereum/go-ethereum/common"
+
+var (
+	// TenantKey is used as key for the tenant identity in the context.ContextWithValue.
+	TenantKey struct{}
+
+	// DummyIdentity to be used until we have identity coming from auth header
+	// TODO(ved): get rid of this once we have multitenancy enabled/
+	DummyIdentity = common.Address([20]byte{1})
```

Comment:
This will be used until we start fetching identity from HTTP calls. This should not break any existing APIs and helps us continue to add multi-tenancy code

Reply:
I hope we would remove this package with it as well..